app/src/main/java/ch/sheremet/dasfoo/rover/android/client/menu/SharedPreferencesHandler.java

```diff
@@ -0,0 +1,102 @@
+package ch.sheremet.dasfoo.rover.android.client.menu;
+
+import android.content.Context;
+import android.content.SharedPreferences;
+import android.preference.PreferenceManager;
+import android.text.TextUtils;
+
+import java.util.MissingFormatArgumentException;
+
+/**
+ * Created by Katarina Sheremet on 6/18/16 4:01 PM.
+ * <p/>
+ * Class is for managing SharedPreferences to get host, port and password.
+ */
+public class SharedPreferencesHandler {
+
+    /**
+     * Context for getting shared preferences.
+     */
+    private final Context mContext;
+
+    /**
+     * Constructor.
+     *
+     * @param context context
+     */
+    public SharedPreferencesHandler(final Context context) {
+        this.mContext = context;
+    }
+
+    /**
+     * Gets host for video.
+     *
+     * @return host
+     * @throws MissingFormatArgumentException if host is empty
+     */
+    public final String getVideoHost() throws MissingFormatArgumentException {
+        return getString(Settings.VIDEO_HOST);
+    }
+
+    /**
+     * Gets port for video.
+     *
+     * @return port
+     * @throws MissingFormatArgumentException if port is empty
+     */
+    public final int getVideoPort() throws MissingFormatArgumentException {
+        return Integer.parseInt(getString(Settings.VIDEO_PORT));
+    }
+
+    /**
+     * Gets grpc host. If host is empty, it tries to return host from video setting.
+     *
+     * @return host
+     * @throws MissingFormatArgumentException if host is empty
+     */
+    public final String getGrpcHost() throws MissingFormatArgumentException {
+        try {
+            return getString(Settings.GRPC_HOST);
+        } catch (MissingFormatArgumentException e) {
+            return getVideoHost();
+        }
+
+    }
+
+    /**
+     * Gets grpc post.
+     *
+     * @return grpc port
+     * @throws MissingFormatArgumentException if post is empty
+     */
+    public final int getGrpcPort() throws MissingFormatArgumentException {
+        return Integer.parseInt(getString(Settings.GRPC_PORT));
```

Comment:
Is it possible to use [`getInt`](https://developer.android.com/reference/android/content/SharedPreferences.html#getInt(java.lang.String, int)) instead?